client/js/_author/components/_index.jsx

```diff
@@ -1,10 +1,23 @@
 import React            from 'react';
+import { connect }            from 'react-redux';
+import * as BankActions       from '../../actions/qbank/banks';
 
-export default class Index extends React.Component {
+
+function select(state) {
+  return {
+    bank: state.banks,
+  };
+}
+export class Index extends React.Component {
```

Comment:
I mean I understand why this is kind of thing is necessary. I think this is fine for now.